frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -421,409 +423,378 @@ class BitBox02 extends Component<Props, State> {
-                                                    onClick={this.createBackup}
-                                                    disabled={creatingBackup || !readDisclaimers}>
-                                                    {t('securityInformation.create.button')}
-                                                </Button>
-                                            </div>
-                                        </div>
-                                        <div className="text-center m-top-large">
-                                            <SwissMadeOpenSource large />
-                                        </div>
-                                    </Step>
+            <Main>
+                { waitDialog && (
+                    <WaitDialog title={waitDialog.title}>
+                        {waitDialog.text ? waitDialog.text : t('bitbox02Interact.followInstructions')}
+                    </WaitDialog>
+                )}
+                { (status === 'connected') ? (
+                    <View
+                        key="connection"
+                        fullscreen
+                        textCenter
+                        withBottomBar
+                        width="600px">
+                        <ViewHeader title={t('button.unlock')}>
+                            <p>{t('bitbox02Wizard.stepConnected.unlock')}</p>
+                        </ViewHeader>
+                        <ViewContent fullWidth>
+                            {attestationResult === false ? (
+                                <Status>
+                                    {t('bitbox02Wizard.attestationFailed')}
+                                </Status>
+                            ) : (
+                                <PasswordEntry />
+                            )}
+                        </ViewContent>
+                    </View>
+                ) : null }
+
+                {(status === 'unpaired' || status === 'pairingFailed') && (
+                    <View
+                        key="failed-pairing"
+                        fullscreen
+                        textCenter
+                        withBottomBar
+                        width="670px">
+                        <ViewHeader title={t('bitbox02Wizard.pairing.title')}>
+                            <p>{t('bitbox02Wizard.stepUnpaired.verify')}</p>
+                        </ViewHeader>
+                        <ViewContent fullWidth>
+                            { status === 'pairingFailed' ? (
+                                <Status type="warning">
+                                    {t('bitbox02Wizard.pairing.failed')}
+                                </Status>
+                            ) : (
+                                attestationResult === false && (
+                                    <Status type="warning">
+                                        {t('bitbox02Wizard.attestationFailed')}
+                                    </Status>
```

Comment:
Only show attestation error if that is the only warning/error, i.e. if the user cancels pairing show the pairingFailed error instead of attestation warning.